src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  --main--color--: #fff;
+  --for--links--: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: var(--main--color--);
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  color: #000;
+  height: 60px;
```

Comment:
don't hardcode height, use vertical paddings

Reply:
this not fixed. you can set `padding: 22px 0;` here instead of height